src/pages/using-spark/guides/writing-meaningful-web-content-with-semantic-html.mdx

```diff
@@ -0,0 +1,250 @@
+
+<span id="examples" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h5"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayFive"
+>
+  Examples
+</SprkHeading>
+
+Here are some examples of emphasized word(s) using the strong element.
+
+- Check your **To Do List** items for new tasks.
+  - Check your `<strong> To Do List </strong>` items for new tasks.
+- You can change your communication preferences on **My Profile**.
+  - You can change your communication preferences on `<strong>My Profile</strong>`.
+- **Your payment is due** on September 22, 2021.
+  - `<strong>Your payment is due</strong>` on September 22, 2021.
+- Remember to bring your **driver’s license** to your closing appointment.
+  - Remember to bring your `<strong>driver’s license</strong>` to your closing appointment.
+
+#### The Strong vs The Bold Element vs Styling Text
+
+- Use the **strong element** `<strong></strong>` for important content, and to communicate urgency.
+- Use the <b>bold element</b> `<b></b>` to bring attention to text without indicating that it's more important.
+  - Use for:
+    - Drawing the reader's attention to the elements contents, which aren't of special importance
+    - Keywords in a summary and for product names in a review
+- Use **CSS** (Cascade Styling Sheet) to style or decorate text, like the <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight"> font-weight style property.</a>
+
+<span id="the-emphasis-element" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h3"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayThree"
+>
+  The Emphasis Element
+</SprkHeading>
+
+Use the **emphasis** element to give "stressed emphasis" to the enclosed text inside the `<em></em>` tags, in the same way verbal stress works.
+
+- By default, web browsers render the contents inside the emphasis tags as <i>italic type.</i>
+
+- A screen reader will pronounce the words inside the **emphasis** element `<em></em>` tags with an emphasis, using verbal stress.
+
+> As a UX Writer, is important to know when to apply **stress emphasis** to text, because it can **change the meaning** of the sentence.
+
+#### Usage
+
+Use for:
+
+- Words that have a stressed emphasis compared to surrounding text.
+  - For example: Our family <em>loves</em> spending summers on the beach.
+- Changing the meaning of a sentence.
+  - For example: We found the perfect house, let's put an offer <em>
+    now
+      </em>!
+- Referencing a specific instance of microcopy within the web page that users can take action on.
+  - For example: <em>
+    Find Brokers Near Me
+      </em> to connect with a broker in your community.
+
+<span id="examples" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h5"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayFive"
+>
+  Examples
+</SprkHeading>
+
+Here are some examples using the em element to give stressed emphasis to some of text.
```

Comment:
We don't need to say "here are examples" when the headline already signaled that this is the example section. We can get rid of this line, as it doesn't add insight.